util/packer/ubuntu-14.04/template.json

```diff
@@ -42,7 +42,8 @@
       "ssh_username": "vagrant",
       "ssh_wait_timeout": "20m",
       "vboxmanage": [
-        ["modifyvm", "{{ .Name }}", "--memory", "1024"]
+        ["modifyvm", "{{ .Name }}", "--memory", "4096"],
+        ["modifyvm", "{{.Name}}", "--cpus", "4"]
```

Comment:
Not a major thing, but I'd probably keep the whitespace consistent with the line above